pkg/broker/broker.go

```diff
@@ -85,16 +88,15 @@ func (b *Broker) Provision(request *osb.ProvisionRequest, _ *broker.RequestConte
 	}
 
 	if namespace == "" {
-		err := errors.New("Cannot provision with empty namespace")
-		glog.Errorln(err)
-		return nil, err
+		klog.V(5).Infof("broker: failed to provision %q with empty namespace", request.InstanceID)
+		return nil, errors.New("Cannot provision with empty namespace")
 	}
 
-	glog.V(5).Infof("Provisioning %s (%s/%s) in %s", request.InstanceID, request.ServiceID, request.PlanID, namespace)
+	klog.V(4).Infof("broker: provisioning request %+v in namespace %q", request, namespace)
 
 	operationName, err := b.Client.Provision(request.InstanceID, request.ServiceID, request.PlanID, namespace, request.AcceptsIncomplete, request.Parameters)
 	if err != nil {
-		glog.Errorln(err)
+		klog.V(5).Infof("broker: failed to provision request %q: %v", request.InstanceID, err)
```

Comment:
Shouldn't we be using 3 or 4 for (user-level) errors? At least, it seems odd to log starting provision more than provision error.

Reply:
True!

Reply:
Fixed with https://github.com/kubernetes-sigs/minibroker/pull/57/commits/05020bc5ad39e314d29af238267f7f19ecb777d1.